tree/dataframe/src/RLoopManager.cxx

```diff
@@ -415,7 +409,7 @@ RLoopManager::RLoopManager(ROOT::RDF::Experimental::RDatasetSpec &&spec)
          // Otherwise, the new friend chain needs to be built using the nomenclature
          // "filename?#treename" as argument to `TChain::Add`
          for (auto j = 0u; j < nFileNames; ++j) {
-            frChain->Add((thisFriendFiles[j] + "?#" + thisFriendChainSubNames[j]).c_str());
+            frChain->Add((thisFriendFiles[j] + "/" + thisFriendChainSubNames[j]).c_str());
```

Comment:
there is a comment two lines up that says "the new friend chain needs to be built using the nomenclature "filename?#treename"".

Either the comment needs to change, or we really need `?#` here. If the file names have already been expanded at this point can we really not use `?#`? I would think that with expanded file names we don't incur in #11483 .

Reply:
We do not expand the friends in advance. We don't really need to. The quickfix as of now here is to use `/` and update the comment.